src/glue.rs

```diff
@@ -32,4 +32,67 @@ impl Glue {
             }
         }
     }
+
+    pub fn select_as_string(&mut self, query: &Query) -> Result<Vec<Vec<String>>> {
```

Comment:
Hmm... it sounds so good to have `select_as_string` function but not sure about the return type.

Is there any special reason to merge labels and values?
![image](https://user-images.githubusercontent.com/2025065/112335441-950ae380-8cff-11eb-8d1f-4f26899ea619.png)

How about simply keeping both?

```rust
Result<(Vec<String>, Vec<Vec<String>>)>
```

Reply:
I was thinking of doing that. Unsure.
I suppose this idea is having just one very makes it more like a CSV.

It this case you don't really _need_ to separate the labels and the rows because you know [0] will always be labels.

Thoughts?

Reply:
A large component to the `select_as_string` idea was that it was kinda for display more than usage.

As an example usage, the user does `select_as_string` then displays it as a table.

I can't really think of many other ways that it would be useful.
Can you?

Reply:
> It this case you don't really need to separate the labels and the rows because you know [0] will always be labels.

This sounds reasonable.
`select_as_string` is for providing convenience to users and in this assumption, it looks good to have current return type. Let's keep the current :) thanks.

Reply:
Awesome. Sounds good.